src/features/CarePackages/DayCare/assets/dayCare.scss

```diff
@@ -30,3 +30,21 @@
         }
     }
 }
+
+.opportunity-summary-line {
+    border-bottom: 2px solid #DEE0E2;
+    height: 0px;
+    width: 100%;
+}
+
+.text-green{
+    color: #00664F;
+}
+
+.text-black {
+    color: #0B0C0C;
```

Comment:
There are stored variables in assets/css/variables.scss to void duplication of things like colours, makes it easier to make site wide changes on colours etc as well